common/rdm/PidStoreHelper.cpp

```diff
@@ -286,6 +288,15 @@ void PidStoreHelper::SupportedPids(
   if (store) {
     store->AllPids(descriptors);
   }
+
+  // Sort PIDs by name
+  sort(descriptors->begin(), descriptors->end(), &CompPidsForSort);
+}
+
+
+bool PidStoreHelper::CompPidsForSort(const PidDescriptor *a,
```

Comment:
Perhaps ComparePidsByName would be clearer?

Do you fancy adding just a basic @brief Doxygen like some of the other bits have too?